modules/bending_energy_diffusion_loss_notes.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<p>It's true that this figure suggests that diffusion loss is invariant to overall image scaling, but the overlapping of the curves is not what suggests it. Rather it's the flatness of the curve that suggests it. This should be rephrased to make it clear that:</p><p> </p><ul><li>the curves are flat, so we see that diffusion loss is invariant to overall image scale regardless of whether normalization is enabled.</li><li class="ql-indent-1">(the curves are not completely flat, but they eventually become flat for larger scales. they fail to be flat for small scales. this is due to the fact that the analysis that was done to determine the correct normalization factor was a <em>continuous</em> analysis, while at small scales we have a <em>discrete</em> sampled image. the error  in the normalization factor is worse when the image is more coarsely sampled, i.e. at smaller scales.)</li><li>the curves overlap, so we see that enabling normalization has no effect on diffusion loss for images with equal dimension.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Thanks for the suggestions! For the second bullet point, just to clarify: since both curves (normalized or unnormalized) fail to be flat at smaller scales (and I think this is true for all curves in this notebook), do we maybe want to say something along the lines of “using central finite difference as an approximation of spatial gradients does not work well at a smaller resolution”? </p>

Reply:
I think that's the reason but maybe also more than that. It's a combination of (a) central finite difference becoming a worse approximation at coarser sampling and (b) we estimate average value of the bending/diffusion loss across the image by summing over all pixels and dividing by the number of pixels $|\Omega|$, and this results in a worse estimate of the true average value at coarser sampling.